5calls/app/src/main/java/org/a5calls/android/a5calls/MainActivity.java

```diff
@@ -172,9 +138,7 @@ private void showStats() {
         Snackbar.make(findViewById(R.id.activity_main), message, Snackbar.LENGTH_LONG).show();
     }
 
-    private void submitZip() {
-        EditText zipEdit = (EditText) findViewById(R.id.zip_code);
-        String code = zipEdit.getText().toString();
+    private void submitZip(String code, EditText zipEdit) {
```

Comment:
If you are passing in the EditText as a param, you shouldn't need to also pass in the code?